marklogic-data-hub/src/main/java/com/marklogic/hub/impl/ScaffoldingImpl.java

```diff
@@ -43,7 +43,7 @@
 
 @Component
 public class ScaffoldingImpl implements Scaffolding {
-    
+
     @Autowired
```

Comment:
still @Autowired?

Reply:
I'd haven't verified that isn't autowired by other Data Hub code. It isn't autowired through any of the OneUI code paths.